tefla/core/eval_metrices.py

```diff
@@ -0,0 +1,772 @@
+"""
+    Evaluation API for evaluation matrices and plots, supports classification and regression both.
+"""
+#pylint: disable-msg=too-many-arguments
```

Comment:
Same lone is repeated here; also its not a good practice to put these disable command at the top of the file;  you can put the command just before the class/function/methods for which you want to disable the check.

Reply:
updated